src/templates/name.pug

```diff
@@ -1,95 +1,118 @@
 extends layouts/layout
 
 block content
-  if (name.info)
-    .contentContainer
-      .card
-        header.card-header
-          p.card-header-title Name Summary
-        .card-content
-          .columns
-            .column
-              .blockFullLabel Name
-              .blockFullDetail= name.info.name
-            .column
-              .blockFullLabel State
-              .blockFullDetail= name.info.state
-            .column
-              .blockFullLabel Claimed
-              .blockFullDetail= name.info.claimed
+  .contentContainer
+    if (name.info)
+        .card
+          header.card-header
```

Comment:
I hope git eventually changes their standards/detection on counting this as a line change...Really makes it hard to read when they act like everything is a change... Anywho, this file looks good.